R/onAttach.R

```diff
@@ -18,20 +18,24 @@
   }
   dev = as.integer(v[1L, 3L]) %% 2L == 1L  # version number odd => dev
   if (!isTRUE(getOption("datatable.quiet"))) {   # new option in v1.12.4, #3489
-    packageStartupMessage("data.table ", v, if(dev)paste0(" IN DEVELOPMENT built ",d,g),
-                          " using ", getDTthreads(verbose=FALSE), " threads (see ?getDTthreads).  Latest news: r-datatable.com")
+    dev_tag = if (dev) gettextf(" IN DEVELOPMENT built %s%s", d, g, domain="R-data.table") else ""
```

Comment:
a bit ugly but `xgettext` ignores the strings passed to `...` of `gettextf` for translation -- even if those strings are `gettext` themselves. so we have to pull this translation out of the `gettextf` call for it to work.

ditto below